urchin_config.toml

```diff
@@ -19,3 +19,11 @@ webserver_port = 8080
 
 # Directory to use for storing uploaded images.
 image_dir = "./images"
+
+links = [
```

Comment:
I like it, but I think we should have a `[navbar]` section here, something like

```toml
[navbar]
page= "/"
links = [
...
]
```

This way we could change navbar links per page. However, don't suggest adding the page in this PR, just put the `[nvabar]` section (by doing an extra inner struct in the config struct), then we can implement extensions in the future 😄

Reply:
Added `[navbar]` as suggested.